instrumentation/que/Rakefile

```diff
@@ -26,3 +26,15 @@ if RUBY_ENGINE == 'truffleruby'
 else
   task default: %i[test rubocop yard]
 end
+
+task :install_test_deps do
+  sh 'bundle exec appraisal install'
+end
+
+task test_que1: :install_test_deps do
+  sh 'bundle exec appraisal que-1.2 rake test'
+end
+
+task test_que2: :install_test_deps do
+  sh 'bundle exec appraisal que-2.1 rake test'
+end
```

Comment:
I think we should remove these:
- running `bundle exec appraisal` is enough to install deps
- running tests can be done with `bundle exec appraisal rake test`, which tests all versions in the `Appraisal` file in series.

Maybe we can just document this in the README for the gem instead?

Reply:
Thanks! I removed the rake tasks and added a section to README but I'd like feedback on that :) https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/205/commits/31d541b7fe781c079fee83c0b805ef241bcace09